pages/index.page.tsx

```diff
@@ -181,18 +181,19 @@ const Home = (props: any) => {
             <div className='lg:w-[650px]  mx-auto my-10 grid grid-cols-1 lg:grid-cols-3 gap-8 justify-items-center '>
               <Link
                 href='/learn/getting-started-step-by-step'
-                className=' flex items-center justify-center rounded border-2 border-white text-white w-[194px] h-[40px] font-semibold  dark:shadow-2xl'
+                className='flex items-center justify-center rounded border-2 border-white hover:bg-blue-700 transition-all duration-500 ease-in-out text-white w-[194px] h-[40px] font-semibold'
               >
                 Getting started
               </Link>
               <Link
                 href='/slack'
-                className=' flex items-center justify-center rounded border-2 border-white text-white  w-[194px] h-[40px] font-semibold dark:border-shadow-white  dark:shadow-2xl'
+                className='flex items-center justify-center rounded border-2 border-white hover:bg-blue-700 transition-all duration-500 ease-in-out text-white  w-[194px] h-[40px] font-semibold'
               >
                 Join Slack
               </Link>
 
-              <div className='flex herobtn items-center justify-center font-semibold w-[194px] h-[40px] rounded border-2 border-white text-white mx-auto  dark:shadow-2xl'>
+
```

Comment:
Hi Vivek, your PR is almost complete just one lint check is failing can you pls. fix it.